source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -649,20 +649,110 @@ msgid "Kafka-WS"
 msgstr ""
 
 #: ../../source/internal-communication.rst:558
+#, fuzzy
```

Comment:
You should correct the ones with the fuzzy marker. Use the poedit for better PO files handling.